cmd/make.go

```diff
@@ -214,6 +214,6 @@ func init() {
 		},
 	}
 
-	makeCmd.PersistentFlags().Uint32Var(&rawArgs.quota, "quota", 0, "specifies the maximum size of the share in gigabytes, 0 means you accept the file service's default quota.")
+	makeCmd.PersistentFlags().Uint32Var(&rawArgs.quota, "quota-gb", 0, "specifies the maximum size of the share in gigabytes, 0 means you accept the file service's default quota.")
```

Comment:
Clarify that it's GiB?

Reply:
Sound OK if I clarify only in the comments?  Alternative, naming it quota-gib seems odd to me somehow. (Not 100% sure why).

Reply:
it could be in the flag description, right?

Reply:
Yes. I've put it in the description.  Which was what I meant, when I foolishly said "comments"

Reply:
Sounds good to me!